src/webapp/js/components/app.js

```diff
@@ -88,7 +88,7 @@ export default class App extends Component {
         </div>
 
         <Alert
-          timeout={5000}
+          timeout="none"
```

Comment:
just remove it - default timeout is 0 which should work. test it pls

Reply:
When I removed or set 0 then not working.

Reply:
ok you convinced me :+1: